### PR TITLE
added mStable_mUSD

### PR DESCRIPTION
**Pull request by Crypto-One-dev**

### CODE REVIEW

File: data/mainnet/tokens.js [hunk 1]
```diff
@@ -5148,6 +5168,14 @@ module.exports = {
       params: [addresses.MATIC.V2.jarvis_AUR_USDC_V2.NewVault, '18', CHAINS_ID.MATIC_MAINNET],
     },
   },
+  MTA: {
+    chain: CHAINS_ID.MATIC_MAINNET,
+    logoUrl: './icons/mta.svg',
+    tokenAddress: addresses.MTA,
```
[the remark this answers is not in the document]
Comment (CryptJS13) [reply data/mainnet/tokens.js:5174]: Also addresses.MTA seems to be undefined anyways

File: data/mainnet/tokens.js [hunk 2]
```diff
@@ -150,6 +150,26 @@ module.exports = {
...
+    apyIconUrls: ['./icons/mta.svg', './icons/wmatic.png'],
+    apyTokenSymbols: ['MTA', 'WMATIC'],
+    displayName: 'mStable_mUSD',
+    tokenAddress: addresses.MATIC.V2.mUSD.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.mUSD.NewVault,
+    priceFunction: { type: GET_PRICE_TYPES.COINGECKO_CONTRACT, params: [addresses.mUSD] },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.MSTABLE,
```
Comment (CryptJS13) [inline data/mainnet/tokens.js:167]: The reduction coming from the performance fee needs to be added in here. On polygon we take a 8% performance fee on rewards

**Review by CryptJS13 — COMMENTED** [review]
Looks pretty good, but got some comments that need to be dealt with.

File: data/constants.js [hunk 3]
```diff
@@ -29,6 +29,7 @@ const VAULT_CATEGORIES_IDS = {
   IDLE: 'IDLE',
   INACTIVE_POLYGON: 'INACTIVE_POLYGON',
   GENOMES: 'GENOMES',
+  MSTABLE: 'mStable',
```
Comment (CryptJS13) [inline data/constants.js:32]: Sorry, this is not what I meant. It needs to be `MSTABLE: 'MSTABLE'` for VAULT_CATEGORIES_IDS, ESTIMATED_APY_TYPES and TRADING_APY_TYPES. Next to that you need to add an input to VAULT_CATEGORIES_NAMES which is `MSTABLE: 'mStable'`

File: data/mainnet/pools.js [hunk 4]
```diff
@@ -4,6 +4,43 @@ const addresses = require('./addresses.json')
...
+    chain: CHAINS_ID.MATIC_MAINNET,
+    id: 'mUSD',
+    type: POOL_TYPES.INCENTIVE,
+    contractAddress: addresses.MATIC.V2.mUSD.NewPool,
+    collateralAddress: addresses.MATIC.V2.mUSD.NewVault,
+    rewardAPY: [],
+    rewardAPR: null,
+    rewardTokens: [addresses.MATIC.miFARM],
+    rewardTokenSymbols: ['miFARM'],
+    stakeAndDepositHelpMessage: `
```
[the remark this answers is not in the document]
Comment (CryptJS13) [reply data/mainnet/pools.js:21]: Sorry, still not making sense. Probably best to refer to `https://mstable.app/#/musd/swap` where users can swap stables to mUSD. Or remove it completely.

File: src/lib/constants.js [hunk 5]
```diff
@@ -63,6 +63,7 @@ const ESTIMATED_APY_TYPES = {
   CONVEX: 'CONVEX',
   BALANCER: 'BALANCER',
   BALANCER_POLYGON: 'BALANCER_POLYGON',
+  MSTABLE: 'mStable',
```
Comment (CryptJS13) [inline src/lib/constants.js:66]: Please change back to MSTABLE: 'MSTABLE'

File: src/lib/constants.js [hunk 6]
```diff
@@ -74,6 +75,7 @@ const TRADING_APY_TYPES = {
   CONVEX: 'CONVEX',
   BELT: 'BELT',
   VENUS: 'VENUS',
+  MSTABLE: 'mStable',
```
Comment (CryptJS13) [inline src/lib/constants.js:78]: Please change back to MSTABLE: 'MSTABLE'

**Review by CryptJS13 — COMMENTED** [review]
Thanks for the updates, some things still need to be changed

**Comment by CryptJS13** [comment]
Looks good! Thanks!

**Review by Cryo45 — APPROVED** [review]

**Pull request merged**